tools-v2/README.md

```diff
@@ -1291,6 +1292,27 @@ Output:
 +------------+-----------+--------+--------+--------+---------+
 ```
 
+####  check server
```

Comment:
Complement the comparison of old and new tools
## Comparison of old and new commands
### curve bs